pom.xml

```diff
@@ -662,12 +690,6 @@ flexible messaging model and an intuitive client API.</description>
         <version>${jersey.version}</version>
       </dependency>
 
-      <dependency>
-        <groupId>net.java.dev.jna</groupId>
```

Comment:
This is needed to pin the dependency version

Reply:
ok, will revert